app/src/organisms/Devices/ProtocolRun/SetupModuleAndDeck/SetupFixtureList.tsx

```diff
@@ -118,7 +119,11 @@ export function FixtureListItem({
   if (configurationStatus !== CONFIGURED) {
     statusLabel = (
       <StatusLabel
-        status={configurationStatus ?? ''}
+        status={
+          configurationStatus === CONFLICTING
+            ? t('location_conflict')
+            : configurationStatus ?? ''
```

Comment:
Do we need to display `''`?

```suggestion
            : configurationStatus ?? ''
```

Reply:
that configurationStatus is possibly undefined but a user should never run into displaying an undefined status.

Reply:
i can write the if/else a bit more elegantly though